frontend/src/content/news.json

```diff
@@ -1,9 +1,14 @@
 {
   "2021": [
+    {
+      "date": "2021-11-09",
+      "title": "Metabolic Atlas 2.1 revamps data overlay",
+      "text": "<b>Metabolic Atlas 2.1</b> is out. In this minor release we focused on improving the usability of the website, either through feature re-implementation, as was the case for the <i>Data overlay</i>, or through independent fixes. Moreover, several small new features were added, for example the <i>Identifier</i> pages. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

Comment:
`Data overlay` has been used at multiple places in this file but with different format, might be good to make it consistent.

Reply:
Thanks, should be resolved in 0986877